### PR TITLE
Created load_data function.

### PR DESCRIPTION
**Pull request by codemasta14**
I created a function that I think should solve all the problems we talked about in our issue. I can clean it up a bit better if this looks good to you so far.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -34,3 +34,4 @@ vignettes/*.pdf
 
 # Shiny token, see https://shiny.rstudio.com/articles/shinyapps.html
 rsconnect/
+.Rproj.user
```
Comment (taltstidl) [inline .gitignore:37]:
```suggestion
```
I think this is already ignored further up this file, so it should not be necessary here.

File: R/load_data.R [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#'@export
+#'
+
+load_data <- function(){
+  
+  path <- rappdirs::user_data_dir("GeyseRdata.tsv","GeyseRdata")
```
Comment (taltstidl) [inline R/load_data.R]: I'm not really sure whether this is a good idea 🤔. I'm generally not a huge fan of software putting files in places where I'm not aware of them, and I'm pretty sure most people will not know to look in their data directory after running this function. I'd vote for a simple temporary file with an option to specify a target folder as a parameter. Thoughts?

File: R/load_data.R [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  
+  if(!file.exists(path)){
```
Comment (taltstidl) [inline R/load_data.R]: While this check ensures that a file is available, it does not offer to either automatically download a more recent version and then overwrite the current file or at least notify the user of the function that a more recent version is available. I believe that's a must for efficient management of the database file 😃.

**Review by taltstidl — CHANGES_REQUESTED** [review]
@codemasta14 A good start 👍, but I have a few comments that I'd like to see addressed. Also, I think @spkaluzny also started with an implementation of this function, so I'd like to have his input, too.

**Comment by codemasta14** [comment]
@TR4Android @spkaluzny I went ahead and made changes to the function, so that it has the functionality you requested.

**Comment by codemasta14** [comment]
Also, it appears that the https://geysertimes.org/archive/ page no longer has a link to the archive. Is it possible that in my writing I crashed it, or just coincidence that it stopped working while I was working on it?

**Comment by EZWrighter** [comment]
> Also, it appears that the https://geysertimes.org/archive/ page no longer has a link to the archive. Is it possible that in my writing I crashed it, or just coincidence that it stopped working while I was working on it?

We just moved geysertimes to a new server and a script wasn't working correctly.  That has been resolved and the archive is now available again.

File: R/load_data.R (hunk 2 above)
Comment (codemasta14) [reply R/load_data.R]: I fixed this and made it so they have the option to specify the location. In the function I also print out where the location is saved to.

File: R/load_data.R (hunk 3 above)
Comment (codemasta14) [reply R/load_data.R]: I went ahead and implemented the changes you suggested.

**Comment by taltstidl** [comment]
@codemasta14 I'll be closing this pull request, unless there are any objections here. All of the functionality is provided in the newer pull request #3.

**Pull request closed without merging**